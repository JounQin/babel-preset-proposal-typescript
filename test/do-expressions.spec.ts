import _ from './do-expressions'
import { execute } from './helpers'

const proposal = 'do-expressions'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_(20)).toBe('big')
  expect(_(0)).toBe('small')
})
