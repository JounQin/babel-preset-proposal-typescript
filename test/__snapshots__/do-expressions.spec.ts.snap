// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`do-expressions 1`] = `"Unexpected token ')'"`;
